<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ScanSetupDvbsWidget</class>
 <widget class="QWidget" name="ScanSetupDvbsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>305</width>
    <height>398</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Scan Setup</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>DVB-S scan setup</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Start frequency [MHz]:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QSpinBox" name="startFrequency">
        <property name="maximum">
         <number>2000000</number>
        </property>
        <property name="value">
         <number>470000</number>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>End frequency [MHz]:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSpinBox" name="endFrequency">
        <property name="maximum">
         <number>2000000</number>
        </property>
        <property name="value">
         <number>826000</number>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Symbol rate [kSymb/s]:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QSpinBox" name="symbolRate">
        <property name="maximum">
         <number>2000000</number>
        </property>
        <property name="value">
         <number>22000</number>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_11">
        <property name="text">
         <string>System</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QComboBox" name="system">
        <property name="sizeAdjustPolicy">
         <enum>QComboBox::AdjustToContents</enum>
        </property>
        <item>
         <property name="text">
          <string>Auto</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>DVB-S</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>DVB-S2</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Modulation:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QComboBox" name="modulation">
        <property name="sizeAdjustPolicy">
         <enum>QComboBox::AdjustToContents</enum>
        </property>
        <item>
         <property name="text">
          <string>Auto</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Q-PSK</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>8-PSK</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>Polarity</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QComboBox" name="polarity">
        <property name="sizeAdjustPolicy">
         <enum>QComboBox::AdjustToContents</enum>
        </property>
        <item>
         <property name="text">
          <string>Auto</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Horiz</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Vert</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="label_12">
        <property name="text">
         <string>Satellite position</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QSpinBox" name="satPos">
        <property name="maximumSize">
         <size>
          <width>47</width>
          <height>20</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Scan step [MHz]:</string>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
       <widget class="QSpinBox" name="scanStep">
        <property name="maximum">
         <number>2000000</number>
        </property>
        <property name="value">
         <number>8000</number>
        </property>
       </widget>
      </item>
      <item row="8" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Number of steps:</string>
        </property>
       </widget>
      </item>
      <item row="8" column="1">
       <widget class="QLabel" name="numberOfSteps">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="ScanSetupCommonWidget" name="commonScanSetup" native="true"/>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Close</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ScanSetupCommonWidget</class>
   <extends>QWidget</extends>
   <header>scansetupcommonwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>clicked(QAbstractButton*)</signal>
   <receiver>ScanSetupDvbsWidget</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>144</x>
     <y>319</y>
    </hint>
    <hint type="destinationlabel">
     <x>144</x>
     <y>169</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
